client/package-lock.json

```diff
@@ -355,7 +355,6 @@
       "version": "1.3.7",
```

Comment:
Revert this file?

Reply:
Evidently we didn't update this after some change to package.json -- this is what I got after running `npm install`. So we should update this.